src/xrAICore/Navigation/level_graph.cpp

```diff
@@ -218,7 +219,7 @@ u32 CLevelGraph::vertex_id(const Fvector& position) const
             if (_y <= position.y)
             {
                 // so, new node is under the specified position
-                if (position.y - _y < position.y - y)
+                if (_y > y)
```

Comment:
This change is not explained, so I can't understand why it's changed.

Reply:
I've tried to simplify the expressions, consider:
```
if (position.y - _y < position.y - y) {
```

assume `position.y` is `A`:

```
const auto A = position.y;
if (A - _y < A - y) {
```

then we can simplify expression using algebra rules (subtract `A` from both sides):

```
if (- _y < - y) {
```

then multiply both sides on `-1`, which reverted expression sign (consider `1 < 2`, than `-1 > -2`):

```
if (-1 * -_y > -1 * -y) {
```

and, finally, simplify expression by partial evaluation:

```
if (_y > y) {
```

Reply:
@Xottab-DUTY If change is too suspicious, and explanation is not well, i can remove the change, ok?

Reply:
@dimhotepus the explanation is very well. That's just me being too neat. I want to test the values in the game, but I won't be able to do that till New Year. Sorry for taking too long on this PR.

So, yes, it would be better to move this particular change to another pull request.
Also, if you can, consider to just remove the commit 553f8ab12f1a107a6c6edee2ee854ce594e0471d via rebase instead of making a reverting commit. Thank you!

Reply:
Not a problem, thank you for review and help! 

I'll remove this one https://github.com/OpenXRay/xray-16/commit/553f8ab12f1a107a6c6edee2ee854ce594e0471d as you suggested.